src/tests/conformance/query/basic.rs

```diff
@@ -0,0 +1,55 @@
+//! Contains test with basic queries.
+//!
+//! Queries and expected replies:
+//!
+//!     - Ping           -> Pong
+
+use crate::{
+    protocol::{
+        codecs::binary::{BinaryMessage, Payload},
+        proto::TmPing,
+    },
+    setup::node::Node,
+    tools::synth_node::SyntheticNode,
+};
+
+#[tokio::test]
+async fn ping() {
+    let mut node = start_node().await;
+    let mut synth_node = start_synth_node().await;
+    synth_node.connect(node.addr()).await.unwrap();
+    let payload = Payload::TmPing(TmPing {
+        r#type: 0,
+        seq: Some(42),
+        ping_time: None,
+        net_time: None,
+    });
+    synth_node.unicast(node.addr(), payload).unwrap();
+    let check = |m: &BinaryMessage| {
+        matches!(
+            &m.payload,
+            Payload::TmPing(TmPing {
+                r#type: 1,
```

Comment:
(this indicates that it's a reply)

Reply:
Might be good to make a comment of this in the test, it's documented in the `.proto` file but anyone reading the test might enjoy this being made explicit here?